File: Perserverance.Shared/Models/SnailyCAD/Citizen.cs
namespace Perserverance.Shared.Models.SnailyCAD
{
    public class Citizen
    {
        public string name { get; set; }
        public string surname { get; set; }
        public object imageId { get; set; }
        public object imageBlurData { get; set; }
        public string id { get; set; }
        public string userId { get; set; }
        public string socialSecurityNumber { get; set; }
        public User user { get; set; }
    }

}

File: nui-source/src/@types/interface/citizen.ts
export interface ICitizen {
  id: string;
  name: string;
  surname: string;
  imageId: string;
  imageBlurData: string;
  socialSecurityNumber: string;
  dateOfBirth: Date;
  gender: string;
  hairColor: string;
  eyeColor: string;
  height: number;
  weight: number;
  address: string;
  postal: string;
  ethnicity: string;
  get fullname(): string;
}
